src/stores/gameList.js

```diff
@@ -52,14 +52,16 @@ export const useGameListStore = defineStore('gameList', {
     joinGame(data) {
       const updatedGame = this.openGames.find((game) => game.id === data.gameId);
       if (updatedGame) {
-        updatedGame.numPlayers++;
+        // updatedGame.numPlayers++;
+        updatedGame.numPlayers = Math.min(2, updatedGame.numPlayers + 1)
```

Comment:
Let's remove the old line and add a ; at the end of this line
```suggestion
        updatedGame.numPlayers = Math.min(2, updatedGame.numPlayers + 1);
```